### ISSUE TITLE
mutil language

### ISSUE THREAD

**Issue by Alice-Thirty** [JaidedAI/EasyOCR#787]
可以把80种语言合再一起训练吗
如中文、英文、日文、韩文、法文等
这样效果会好?

**Comment by rkcosmos** [comment]
yes, it is possible to train many languages together. For example, we trained any language with English. The performance can be degraded if you have too many characters in one model.

**Comment by ShankarParimi** [comment]
instead can there be a pre-processing model that can determine the language present in the image?
